packages/mobile/src/hooks/useLoadOfflineTracks.ts

```diff
@@ -57,11 +58,16 @@ export const useLoadOfflineData = () => {
     for (const collectionId of offlineCollections) {
       try {
         if (collectionId === DOWNLOAD_REASON_FAVORITES) {
-          dispatch(addCollection({ collectionId, isFavoritesDownload: false }))
+          dispatch(
+            completeCollectionDownload({
+              collectionId,
+              isFavoritesDownload: false
```

Comment:
this is a special case because the favorites collection itself is not a collection within favorites, so we track it at the top level with collections that were individually marked for download. Kinda confusing though =\ Adding a comment at least